code/part-two/processor/actions/create_collection.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { NAMESPACE } = require('../utils/constants');
+const { hash, encode, reject } = require('../utils/helpers');
+
+
+module.exports = (context, publicKey) => {
+  const address = NAMESPACE + '01' + hash(publicKey, 62);
```

Comment:
Just wondering why you didn't use es6 template strings. Don't mind either way jw.

Reply:
Just trying to cut down on ES6 syntax (other than let, const and arrow functions). If @therobinkim is of the opinion that template strings won't be an impediment to students, I will def use them.

Reply:
@delventhalz 👍 for template strings!

Reply:
After looking at it as a template string, I have decided that this is a case where they actually make things less readable not more, and have left it as a concatenated string.

```javascript
`${NAMESPACE}01${hash(publicKey, 62)}`
```